Amazon.IonObjectMapper/IonSerializer.cs

```diff
@@ -371,6 +386,17 @@ public object Deserialize(IIonReader reader, Type type, IonType ionType)
                 return null;
             }
 
+            if (type != null) 
+            {
+                var customSerializerAttribute = type.GetCustomAttribute<IonSerializerAttribute>();
+
+                if (customSerializerAttribute != null) {
+                    var customSerializerFactory = (IIonSerializerFactory)Activator.CreateInstance(customSerializerAttribute.Factory);

```

Comment:
We need to look for a `Serializer` here as well.

Reply:
Added.